lib/shared/config/load-files.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+var copyProps = require('copy-props');
+var path = require('path');
+
+function loadConfigFiles(configFiles, configFileOrder) {
+  var config = {};
+
+  configFileOrder.forEach(function(key) {
+    var filePath = configFiles[key];
+    if (!filePath) {
+      return;
+    }
+
+    copyProps(require(filePath), config, function(value, name) {
+      if (name === 'flags.gulpfile') {
+        return path.resolve(path.dirname(filePath), value);
```

Comment:
Oh interesting, this is making the path in the config file relative its location.